cloudsql-mysql-plugin/src/e2e-test/resources/errorMessage.properties

```diff
@@ -1,9 +1,9 @@
-errorMessageInvalidSourceDatabase=SQL error while getting query schema: Error: Unknown database 'invalidDatabase', SQLState: 42000, ErrorCode: 1049
+errorMessageInvalidSourceDatabase=errorMessage: SQL Exception occurred
```

Comment:
I still see generic error messages which is bad.

Reply:
The earlier error message is changed as per the new code, and hence as per the latest code updated the error message not include the dynamic/privacy data present in error msgs here it contains.

Reply:
As suggested, updated the error message.